src/_data/contactless/toc.yml

```diff
@@ -0,0 +1,66 @@
+          - text: Cal-ITP will provide the resource materials and technical support you need to navigate modernizing your fleet to accept contactless payments. To help us better assist you, we ask that you complete an <a href="https://www.camobilitymarketplace.org/contactless-support">interest form</a> to request Cal-ITP support. With this information, we can determine how best to support you with developing a project scope, getting buy-in from internal stakeholders, engaging with vendors, and implementing your new fare payments system.
+          - text: After you complete the form, we welcome you to browse the Contactless Payments MSA Information Packet (this guide) to understand the overall process and your next steps.
+          - title: What else do I need to know about contactless payments?
+          - text: If you’d like to learn more about contactless payments or need an informational presentation to share with your staff or board of directors, please review this Contactless Payments Introduction (PDF).
+      - title: What to know about MSAs?
+        url: "what-to-know-about-msas"
+        paragraphs:
+          - title: What are MSAs?
+          - text: The California Department of General Services (DGS), in collaboration with Cal-ITP, conducted a Request for Proposals (RFP) that established six competitively awarded Master Service Agreements (MSAs), contracts that allow U.S. public transportation providers to purchase directly from California’s DGS without further competitive bidding.
+          - text: The MSAs allow any local government entity that operates public transportation services (“transit providers”) to purchase goods and services off of California’s MSAs. Out-of-state transit providers—including but not limited to members of the Oregon Cooperative Procurement Program (ORCPP) and the Washington State Purchasing Cooperative (WSPC)—can also participate. Each transit provider should determine whether use of the MSAs is consistent with its procurement policies and regulations. California’s MSAs are generally known as State Purchasing Schedules in other states.
+          - text: Projects resulting from these MSAs may be funded through grants from the Federal Transit Administration (FTA), and/or the State of California, California Department of Transportation (Caltrans), and other sources of local and state public funding. Check with your funding agencies.
+      - title: GTFS Compliance
+        url: "gtfs-compliance"
+        paragraphs:
+          - text: General Transit Feed Specification (GTFS) is the global standard for providing up-to-date transit information. We recommend creating an action plan for becoming GTFS-ready before going contactless, as the contactless payment hardware and software services vendors you select can help you use this data for reporting purposes plus help your customers see your real-time vehicle location and fare information on journey-planning smartphone maps and apps.
```

Comment:
Should `General Transit Feed Specification (GTFS)` link to https://dot.ca.gov/cal-itp/california-minimum-general-transit-feed-specification-gtfs-guidelines? @o-ram is this the most correct link? This link says `Version 2.1 – Finalized March 25, 2022`